pyqtgraph/exporters/PrintExporter.py

```diff
@@ -45,7 +45,11 @@ def export(self, fileName=None):
         #res = printer.resolution()
         sr = self.getSourceRect()
         #res = sr.width() * .4 / (self.params['width'] * 100 / 2.54)
-        res = QtGui.QDesktopWidget().physicalDpiX()
+        try:
```

Comment:
Probably no need for a try/except now that we've killed off Qt4, just use `QtGui.QGuiApplication.primaryScreen().physicalDotsPerInchX()`

Reply:
ok done